content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+  ([Issue #27566](https://github.com/istio/istio/issues/27566))
+
+- **Fixed** istioctl wait now tracks resource's metadata.generation field, rather than metadata.resourceVersion.
+Command line arguments have been updated to reflect this.
+  ([Issue #28797](https://github.com/istio/istio/issues/28797))
+
+- **Fixed** a bug where locality weights are only applied when outlier detection is enabled.  ([Issue #28970](https://github.com/istio/istio/issues/28970))
+
+- **Fixed** namespace shorthand flag missing in dashboard subcommand.
+  ([Issue #28970](https://github.com/istio/istio/issues/28970))
+
+- **Fixed** Prevent goroutine leak when using alpha status feature.
+  ([Issue #29275](https://github.com/istio/istio/issues/29275))
+
+- **Fixed** an bug if global sidecar is defined in root namespace, no xDS pushes happen when cluster scoped configs(EnvoyFilter, AuthorizationPolicy, RequestAuthentication) changes.
+  ([Issue #29414](https://github.com/istio/istio/issues/29414))
+
+- **Fixed** a bug where DNS agent preview produces malformed DNS responses
+  ([Issue #29681](https://github.com/istio/istio/issues/29681))
+
+- **Fixed** Modifying multi-network gateway configuration via Services with the label `topology.istio.io/network` will
+now cause endpoints to be updated on all proxies.
+  ([Issue #30054](https://github.com/istio/istio/issues/30054))
+
+- **Fixed** revision is not applied to the scale target reference of HorizontalPodAutoscaler when helm values for hpa are specified explicitly.
+  ([Issue #30203](https://github.com/istio/istio/issues/30203))
+
+- **Fixed** dashboard controlz could not port forward to istiod pod.
+  ([Issue #30208](https://github.com/istio/istio/issues/30208))
+
+- **Fixed** installation issue in which `--readiness-timeout` flag is not honored.
+  ([Issue #30221](https://github.com/istio/istio/issues/30221))
+
+- **Fixed** namespace isn't resolved correctly in VirtualService delegation's short destination host.
+  ([Issue #30387](https://github.com/istio/istio/issues/30387))
+
+- **Fixed** an issue that caused very high memory usage with a large number of `ServiceEntries`.
+  ([Issue #25531](https://github.com/istio/istio/issues/25531))
+
+- **Fixed** an issue to allow nested JSON structure in `meshConfig.accessLogFormat`.
+  ([Issue #28597](https://github.com/istio/istio/issues/28597))
+
+- **Fixed** a bug where load assignments were added to passthrough subsets resulting in Envoy rejecting those subset clusters.
+  ([Issue #25691](https://github.com/istio/istio/issues/25691))
+
+- **Improved** the expose istiod example to use TLS passthrough for multicluster or external Istiod
+**Added** an expose istiod example to use TLS termination for multicluster or external Istiod  ([Issue #27976](https://github.com/istio/istio/issues/27976))
+
+- **Improved** 'verify-install' detects Istio injector without control plane.
+  ([Issue #29607](https://github.com/istio/istio/issues/29607))
```

Comment:
We should lump both `verify-install` commands close together.

Reply:
One is in `Added` and the other is in the `Improved` section. As we have sorted this list alphabetically I think they will be displayed separately.